teachers_digital_platform/css/organisms/filter-panel.less

```diff
@@ -0,0 +1,48 @@
+// Filter panel
+
+.o-filter-panel {
+    .tdp-activity-search & {
+        background-color: #f7f8f9;
+        border: 1px solid #ccc;
```

Comment:
Please replace the color value with the `@gray-20` variable.

Reply:
Done.